src/animation.jl

```diff
@@ -80,7 +80,9 @@ function buildanimation(animdir::AbstractString, fn::AbstractString;
 
     catch err
         warn("""Tried to create gif using convert (ImageMagick), but got error: $err
-            ImageMagick can be installed by executing `Pkg.add("ImageMagick")`
+            ImageMagick needs to be installed in julia and on your machine:
+              1) in julia>> `Pkg.add("ImageMagick")`
+              2) on your machine (in terminal run): sudo apt install imagemagick
```

Comment:
Theoretically it shouldn't be needed on any platform. Did you experience something different, @huckl3b3rry87?